wsclient/src/test/java/org/hpccsystems/ws/client/WsDFUClientTest.java

```diff
@@ -363,4 +367,14 @@ public void searchFileTest() throws Exception
         assertEquals(2,results.size());
         assertEquals(fname2, results.get(0).getFileName());
     }
+
+    @Test
+    public void getClustersTest() throws Exception {
+        String lzfile = System.getProperty("lztestfile");
```

Comment:
If the filename is not provided, we should provide a fallback, such as BaseRemoteTest.DEFAULTHPCCFILENAME.
Also, the property name confuses me a bi, lztestfile (landing zone) implies a file in the landing zone, not a logical file that's been sprayed to the cluster.